new/LIT7103PrayOTprophets.xml

```diff
@@ -63,47 +64,85 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <div type="bibliography">
                 <bibl><ptr target="bm:Strelcyn1949PriereFalacha"/><citedRange unit="page">64-75</citedRange></bibl>
             </div>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="dc:relation" active="LIT7103PrayOTprophets" passive="LIT1828Mahale"><desc>The entire collection
+                        is possibly related to <ref type="work" corresp="LIT1828Mahale"/> and the single prayers may be related to, or renarrations of, 
+                        the songs of the prophets otherwise contained in <ref type="work" corresp="LIT1828Mahale"/>.</desc></relation>
+                    <relation name="dc:relation" active="LIT7103PrayOTprophets" passive=" LIT2277salotu"><desc>The prayer attributed
+                        to Moses is possibly related to <ref type="work" corresp=" LIT2277salotu"/>.</desc></relation>
```

Comment:
Not the entire work 7103 is related, these relations belong to parts only and derived IDs must be used there (and possibly corresp), to make clea which part links to which canticle. No blank spaces in links (quotation marks)